src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -694,19 +694,6 @@ describe('Form Model', () => {
         )
       })
 
-      it('should set authType to NIL when given authType is MyInfo', async () => {
```

Comment:
To check if we already have one test case that tests `should set authType to MyInfo when given authType is MyInfo`

Reply:
[847ea1f](https://github.com/opengovsg/FormSG/pull/6870/commits/847ea1f109f67f7833d17ddbb72fedb4d7851108)